src/effects/VST3/internal/PlugFrame.cpp

```diff
@@ -0,0 +1,31 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file PlugFrame.cpp
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
+
+**********************************************************************/
+
+#include "PlugFrame.h"
+
+internal::PlugFrame::PlugFrame(wxWindow* window)
+   : mWindow(window)
+{
+   FUNKNOWN_CTOR
```

Comment:
Should there be a destructor too that uses the FUNKNOWN_DTOR macro for some debug checking?

Reply:
Won't harm